ttnn/cpp/pybind11/operations/__init__.hpp

```diff
@@ -120,6 +121,9 @@ void py_module(py::module& module) {
     auto m_conv2d = module.def_submodule("conv2d", "conv2d operation");
     conv::conv2d::py_bind_conv2d(m_conv2d);
 
+    auto m_conv_transpose2d = module.def_submodule("conv_transpose2d", "conv_transpose2d operation");
```

Comment:
@mywoodstock , can you please revisit this part? I think there should be one module for convs and this should just be an op in that module.

Reply:
@sankarmanoj-tt please consider binding into the same module as defined on line 121. just call your registration into the same module.

Reply:
Changed to a single py module.